h2o-core/src/main/java/water/fvec/Frame.java

```diff
@@ -253,7 +253,8 @@ public String name(int i) {
   /** The array of keys.
    * @return the array of keys for each vec in the frame.
    */
-  public Key[] keys() { return _keys; }
+  public Key<Vec>[] keys() { return _keys; }
+  public Iterable<Key<Vec>> keysList() { return Arrays.asList(_keys); }
```

Comment:
I do not understand as well - the commit changed API of method `untrack` to accept `Iterable`, but suddenly it does not accept output of method `Frame#keys`. And as a result we have another public method which is not tested and which is not commented. 

It seems for me like a change only for having a change.